ontoportal

```diff
@@ -0,0 +1,238 @@
+
+		echo "[+] Adding some envirement variables for the API"
+		file_path=".env"
+		echo >> "$file_path"
+		echo "REDIS_GOO_CACHE_HOST=redis-ut" >> "$file_path"
+		echo "REDIS_HTTP_CACHE_HOST=redis-ut" >> "$file_path"
+		echo "REDIS_PERSISTENT_HOST=redis-ut" >> "$file_path"
+		
+		echo "[+] Running api script"
+		
+		run_api
+		if [ $? -ne 0 ]; then
+			echo "[-] Error in run_api function. Exiting..."
+			exit 1
+		fi
+
+		echo "[+] The API is running successfully."
+	}
+
+	run_ui_default_mode(){
+		echo "[+] Getting compose file for UI"
+		curl -sS -L https://raw.githubusercontent.com/ontoportal-lirmm/bioportal_web_ui/development/docker-compose.yml -o docker-compose_ui.yml
+		echo >> .env
+		curl -sS -L https://raw.githubusercontent.com/ontoportal-lirmm/bioportal_web_ui/development/.env.sample >> .env
+		
+		sed -i 's/^API_URL=http:\/\/localhost:9393$/API_URL=http:\/\/api-service:9393/' .env
+		sed -i 's/^API_KEY=$/API_KEY=72c72cba-ad45-4785-b94e-483fa55cdddb/' .env
+
+		echo "[+] Changing lines in docker-compose_ui.yml"
+		file_path="./docker-compose_ui.yml"
+		sed -i 's/- .:\/app/- app_web:\/app/' "$file_path"
+		sed -i 's/retries: 3/retries: 100/' "$file_path"
+		sed -i '/^volumes:/a \ \ app_web:' "$file_path"
+		
+		echo "[+] Running ui script"
+		run_ui
+		if [ $? -ne 0 ]; then
+			echo "[-] Error in run_ui function. Exiting..."
+			exit 1
+		fi
+
+		echo "[+] The UI is running successfully."
+	}
+
+	run_api_default_mode
+	run_ui_default_mode
+}
+
+run_personlized_mode(){
+	source .env
+	echo "[+] Getting compose file from: $ORGANIZATION_NAME$COMPOSE_API_FILE_PATH"
+	eval "curl -sS -L https://raw.githubusercontent.com/$ORGANIZATION_NAME$COMPOSE_API_FILE_PATH -o docker-compose_api.yml"
+	echo "[+] Changing some lines"
+	file_path="./docker-compose_api.yml"
+	sed -i 's#- \.:\/srv\/ontoportal\/ontologies_api#- app_api:/srv/ontoportal/ontologies_api#' "$file_path"
+	sed -i 's#./test/solr/configsets:/configsets:ro#./ontologies_linked_data/test/solr/configsets/:/configsets:ro#' "$file_path"
+	sed -i '/^volumes:/a \ \ app_api:' "$file_path"
+	echo "[+] Running api script"
+	run_api
+	
+	
+	echo "[+] Getting compose file from: $ORGANIZATION_NAME$COMPOSE_UI_FILE_PATH"
+	eval "curl -sS -L https://raw.githubusercontent.com/$ORGANIZATION_NAME$COMPOSE_UI_FILE_PATH -o docker-compose_ui.yml"
+	echo "[+] Changing some lines"
+	file_path="./docker-compose_ui.yml"
+	sed -i 's/- .:\/app/- app_web:\/app/' "$file_path"
+	sed -i 's/retries: 3/retries: 100/' "$file_path"
+	sed -i '/^volumes:/a \ \ app_web:' "$file_path"
+	echo "[+] Running ui script"
+	run_ui
+}
+
+clean(){
+  echo "[+] Cleanning the server"
+  echo "[+] Removing ontologies_linked_data"
+  rm -rf ontologies_linked_data
+  echo "[+] Removing docker compose files"
+  rm -f docker-compose*
+  echo "[+] Removing All containers/volumes/networks"
+  docker container rm -f $(docker ps -aq) 2>/dev/null; docker volume rm $(docker volume ls -q) 2>/dev/null ; docker network prune -f
```

Comment:
this clean needs to affect only images/containers related to the current running ontoportal, not removing everything.
As in a server, you may have multiple applications using docker.
Also, the volumes of the database, need to be persistent, we don't want to remove them on each deployment or start.